common/browsered/field-validation-checks.js

```diff
@@ -12,8 +12,10 @@ const validationErrors = {
   currency: 'Choose an amount in pounds and pence using digits and a decimal point. For example “10.50”',
   phoneNumber: 'Must be a 11 digit phone number',
   validEmail: 'Please use a valid email address',
-  isHttps: 'URL must begin with https://',
-  isBelowMaxAmount: `Choose an amount under £${MAX_AMOUNT.toLocaleString()}`
+  https: 'URL must begin with https://',
+  belowMaxAmount: `Choose an amount under £${MAX_AMOUNT.toLocaleString()}`,
+  sortCode: `Sort codes must contain 6 digits`,
+  accountNumber: `Account numbers must contain 8-10 digits`
 }
 
 exports.isEmpty = function (value) {
```

Comment:
Minor: I know it was already like that, but could we use `module.exports` instead of `exports`?

Reply:
Yeah sure